src/app/page/MissionFeedback/Steps.js

```diff
@@ -0,0 +1,173 @@
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import React, { useState } from "react";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import { Button, H1, Body1 } from "../../component";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2, 0),
+  },
+  DeliveryImage: {
+    width: "100%",
+    marginBottom: theme.spacing(2),
+  },
+  Heading: {
+    margin: theme.spacing(0, 2, 2, 2),
+    color: theme.color.deepPurple,
+  },
+  Description: {
+    margin: theme.spacing(0, 2, 2, 2),
+    textAlign: "left",
+  },
+  Button: {
+    margin: theme.spacing(0, 1),
+  },
+  FeedbackInputFeild: {
+    width: `calc(100% - ${theme.spacing(4)}px)`,
+    marginBottom: theme.spacing(2),
+  },
+  Icon: {
+    height: "48px",
+    width: "48px",
+  },
+}));
+
+const Step = {
+  CONFIRM_DELIVERY: "confirmDelivery",
+  FEEDBACK: "feedback",
+  FEEDBACK_WITH_SUCCESS: "feedbackWithSuccess",
+  ACK: "ack",
+};
+
+const DeliveryConfirmation = ({ deliveryImage, setStep }) => {
+  const classes = useStyles();
+  const onClickYes = () => {
+    setStep(Step.FEEDBACK_WITH_SUCCESS);
+  };
+  const onClickNo = () => {
+    setStep(Step.FEEDBACK);
+  };
+  return (
+    <Grid container direction="column" alignItems="center" className={classes.root}>
+      <VisibilityIcon color="primary" className={classes.Icon} />
+      <H1 className={classes.Heading}>Look it's here!</H1>
+      <img src={deliveryImage} alt="Delivery confirmation" className={classes.DeliveryImage} />
+      <Body1 className={classes.Description}>Did you get everything you needed?</Body1>
+      <Box>
+        <Button className={classes.Button} onClick={onClickYes}>
+          Yes
+        </Button>
+        <Button color="secondary" onClick={onClickNo} className={classes.Button}>
+          No
+        </Button>
+      </Box>
+    </Grid>
+  );
+};
+
+const FeedbackForm = ({
+  handleChange,
+  handleSubmit,
+  placeholder,
+  required,
+  submitBtnText,
+  value,
+}) => {
+  const classes = useStyles();
+  const [error, setError] = useState(false);
+  const onSubmit = () => {
+    if (required && !value) {
+      setError(true);
+    } else {
+      handleSubmit();
+    }
+  };
```

Comment:
Is this OK for checking empty input? The form just had one text field so kept it simple.